docs/guides/operator-guide-aws-marketplace.md

```diff
@@ -119,7 +119,7 @@ The following table lists all resources that are created during the [deployment]
 
 Here's what you can customize during or after the [deployment](#deployment):
 
-- VPC: You can either set up a new VPC and subnets or use existing ones.
+- VPC: Existing VPC and VPC Subnet IDs need to be specified.
```

Comment:
Better to be active than passive. Suggest:

- VPC: You must specify the VPC and also any existing VPC Subnet IDs.